app/routes.php

```diff
@@ -30,6 +29,9 @@
     $app->put('/hooks/donation/{donationId:.{36}}', Hooks\DonationUpdate::class)
         ->add(DonationHookAuthMiddleware::class);
 
+    // Unauthenticated as Stripes SDK handles signature verification
```

Comment:
@tbg-jasonfung I would just tweak this comment to point out that the Action _is_ authenticated, just another way - middleware isn't the only way to do auth.